server.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
```

Comment:
Consider adding error handling middleware to handle any unexpected errors that may occur during the execution of your server. This can help improve the robustness of your server. [important]